### ISSUE TITLE
Feature Request: Appimage support

### ISSUE THREAD

**Issue by eugenialoli** [FontManager/font-manager#400]
I'm on the Debian side of the world, and there's no easy way to use the app, apart from using the flatpak. I'd prefer a .deb, but even a universal appimage would work, as I think it has fewer limitations than flatpaks. Thx!

**Comment by fccfc5d245c** [comment]
Yes to this. The .deb we Debian users have on Discover is outdated (0.8.8-2) and as buggy as the Flatpak. I'd also prefer an updated .deb, but if Appimage is an improvement, it would be very welcome.

**Comment by JerryCasiano** [comment]
Sorry but aside from the fedora and debian folders which are present in the repo and used for testing I don't do any packaging.

Anything else is up to interested third parties.
